install/ks/centos7/ksintegralstor.cfg

```diff
@@ -599,18 +599,17 @@ else
   echo "Non dell hardware. Exiting..."
 fi
 
-### Grub and other file modification to show title/name Integralstor Unicell instead Centos 7.2 ###
+### Grub and other file modification to show title/name Integralstor instead Centos 7.2 ###
 cd /etc
-rm -f issue centos-release 
-/usr/bin/wget -c http://192.168.1.150/netboot/distros/centos/6.6/x86_64/integralstor/v1.0/issue
-/usr/bin/wget -c http://192.168.1.150/netboot/distros/centos/6.6/x86_64/integralstor/v1.0/centos-release
-cd /usr/share/plymouth/themes/text
-rm -f text.plymouth
-/usr/bin/wget -c http://192.168.1.150/netboot/distros/centos/6.6/x86_64/integralstor/v1.0/text.plymouth
-dracut -f
+mv issue issue.bak
```

Comment:
It will be having OS name, by defaul like 'centos 6.6'. Renaming is may required in case if we want to know previous entry.